lib/generamba/cli/gen_command.rb

```diff
@@ -38,6 +39,8 @@ def gen(module_name, template_name)
 
       rambafile = YAML.load_file(RAMBAFILE_NAME)
 
+      print_parameters_in_table(rambafile, module_name)
```

Comment:
I think that you can implement it more beautifully using ruby mixins. Just define a new class, smth like TablePrinter, implement this method and include it in all the commands.